classes/class-query.php

```diff
@@ -305,20 +227,19 @@ public function query( $args ) {
 		 */
 		$query = apply_filters( 'wp_stream_db_query', $query, $args );
 
+		$result = array();
 		/**
 		 * QUERY THE DATABASE FOR RESULTS
 		 */
-		$results = $wpdb->get_results( $query ); // @codingStandardsIgnoreLine $query already prepared
-
-		// Hold the number of records found
-		$this->found_records = absint( $wpdb->get_var( 'SELECT FOUND_ROWS()' ) );
+		$result['items'] = $wpdb->get_results( $query ); // @codingStandardsIgnoreLine $query already prepared
```

Comment:
We should change the query, where it's defined above, to:
```
		$query = "SELECT SQL_CALC_FOUND_ROWS %1$s
		FROM %2$s
		%3$s
		WHERE 1=1 %4$s
		%5$s
		%6$s";
```

And then, prepare properly on this line.

I realise that this isn't introduced by you, but it seems like an easy enough fix to include.